roles/openshift_grafana/tasks/main.yaml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Create grafana namespace
+  command: oc create namespace grafana
+
+- name: Switch to grafana project
+  command: oc project grafana
+
+- name: Templatize grafana yaml
+  template: src=grafana-ocp.yaml dest=/tmp/grafana-ocp.yaml
+
+- name: load the grafana taml
+  command: oc new-app -f /tmp/grafana-ocp.yaml
+
+- name: Wait to grafana be running
+  command: oc get pod |grep grafana |awk '{print $3}'
+  register: grafana_pod
+  failed_when: grafana_pod.stdout = 'ImgError'
+  retries: 12
+  delay: 5
+  until: grafana_pod.stdout == 'Running'
+
+- name: Get oc token
+  command: oc sa get-token prometheus -n kube-system
+  register: token
+
+- name: Get oc route
+  command: oc get route |grep grafana |awk '{print $2}'
+  register: route
```

Comment:
https://github.com/openshift/openshift-ansible/blob/master/roles/lib_openshift/library/oc_obj.py

state: list
name: *grafana-route-name*
namespace: *grafana namespace*
kind: route

Reply:
Followed by check for length of results list.